ui/app/src/matrix-store.ts

```diff
@@ -465,12 +466,20 @@ export class MatrixStore {
     }
 
     const renderers = await gui.appletRenderers(
-      weServices,
-      [{ weInfo: this.getWeGroupInfo(weGroupId)!, appInfo }],
-      this.appWebsocket,
       appletAppAgentWebsocket,
+      weServices,
+      [{ neighbourhoodInfo: this.getWeGroupInfo(weGroupId)!, appInfo }],
     );
 
+    this._matrix.update((matrix) => {
```

Comment:
Can we have a comment on this code to explain what it's doing? It looks like a 'recipe'-based cache update handling thing (a-la [immer](https://immerjs.github.io/immer/#with-immer)), but I'm not sure that's right.

Maybe it's injecting Resource rendering `CustomElement` classes into some registry for each Applet?

Reply:
Will do. Basically, I am currently storing an instance of the applet views in the AppletInstanceInfo class:
```
export interface AppletInstanceInfo {
  appletId: EntryHash; // hash of the Applet entry in the applets zome of the group's we dna
  appInfo: AppInfo; // InstalledAppInfo
  applet: Applet;
  federatedGroups: DnaHash[];
  appAgentWebsocket?: AppAgentClient;
  views?: AppletRenderers;
}
```
As it seemed like the appropriate place for any applet related info to be stored in the matrix store.

Reply:
and for reference, the matrix store models group & applet data as such:
```
  private _matrix: Writable<DnaHashMap<[WeGroupData, AppletInstanceInfo[]]>> = // We Group DnaHashes as keys
  ```